src/Text/Ginger/Run.hs

```diff
@@ -156,6 +156,8 @@ defaultScope =
     , ("greaterEquals", fromFunction gfnGreaterEquals)
     , ("int", fromFunction . unaryFunc $ toGVal . fmap (Prelude.truncate :: Scientific -> Int) . asNumber)
     , ("int_ratio", fromFunction . variadicNumericFunc 1 $ fromIntegral . intRatio . Prelude.map Prelude.floor)
+    , ("is_lt", fromFunction gfnIsLt)
+    , ("<", fromFunction gfnIsLt)
```

Comment:
The `<` alias is probably going to be problematic, and also unnecessary, unless you want to support syntax like `{% if foo|<(bar) %}` or `{% if <(foo,bar) %}`; the plain old `<` operator is already taken care of as syntax sugar built into the parser itself: `{% if foo < bar %}` desugars to `{% if lt(foo, bar) %}`, no aliases needed here.